varcode/effect_collection.py

```diff
@@ -262,3 +263,26 @@ def gene_counts(self):
         for effect in self:
             counter[effect.gene_name] += 1
         return counter
+
+    def to_dataframe(self):
+        """Build a dataframe from the effect collection"""
+        def row_from_effect(effect):
+            row = {}
+            row['chr'] = effect.variant.contig
+            row['position'] = effect.variant.start
```

Comment:
might make sense to be clear here about the coordinate system, e.g. call it `inclusive_start`. Just a suggestion, feel free to ignore